packages/packages.csv

```diff
@@ -15,6 +15,7 @@ dotnet-sql-cache,ship
 dotnet-test-xunit,ext
 dotnet-user-secrets,ship
 dotnet-watch,ship
+Libuv,ship-ext
```

Comment:
Updated. Publishing Libuv and Sqlite to `ship-ext`. I'll speak to @joeloff and @phenning about picking up from this directory in addition to `ship` and `ext` for templates and tooling.